tests/regression/test_star_pc.py

```diff
@@ -162,7 +162,7 @@ def test_star_equivalence(problem_type):
     star_its = star_solver.snes.getLinearSolveIterations()
 
     try:
-        from tinyasm import _tinyasm as tasm
+        from tinyasm import _tinyasm as tasm  # noqa
```

Comment:
You can be more specific in the warning you squash I think. Otherwise this is fine.